environment_setup/setup_software.sh

```diff
@@ -200,8 +204,11 @@ host_software_packages=(
     clang-format
     protobuf-compiler
     libprotobuf-dev
+    nodejs
+    yarn
```

Comment:
We should add `nodejs` (and `npm`?) as `<build_depend>`'s in `corner_kick/package.xml`. Yarn is not yet a rosdep, but we should add a `TODO` here to open a PR to add `yarn` as an official rosdep (to here: https://github.com/ros/rosdistro/blob/master/rosdep/base.yaml), and then do that if we end up going with this option.

Reply:
Would still need to add the PPA in the setup script in order to get the latest version of node in Ubuntu 16.04.

In addition, yarn is not available on apt by default so I wonder if this would prevent us from adding it to the official rosdep.